README.md

```diff
@@ -31,6 +31,15 @@ Your password has been pwned 2 times.
 >>>
 ```
 
+on cli:
+
+```console
+$ pwned check
+Password: ****
+Repeat for confirmation: ****
```

Comment:
i just check this once again. the idea is that inputted password should never be printer on this subcommand. that is why i use click https://click.palletsprojects.com/en/7.x/options/#password-prompts . with that feature from click, password input wiill be asked twice and will not be printed out on screen

this doc also wrong because after that check, there is no mask input. so actual input will be seen like this: 

```console
$ pwned check
Password: 
Repeat for confirmation: 
```
but that didn't give user glance about what todo. other alternative is to give coment, for example

```console
$ pwned check
Password: <PASSWORD>
Repeat for confirmation: <PASSWORD>
```
but that also look different than actual thing on screen

Reply:
According to docs there are parameters `hide_input` and `confirmation_prompt` if you add decorator `@click.option` instead of `@click.password_option`. Did you experiment with these also?

See: https://click.palletsprojects.com/en/7.x/options/#password-prompts

Reply:
i just found this reply.

previous goal is to echo asterisk character when user give input. but as seen here https://stackoverflow.com/q/35805078/1766261 no easy way to make those.

i also try `click.option` and see if that decorator can give that feature, but failed.

so imo it is best to keep it simple with `click.password_option` and just write the documentation like that so user know what to input when prompt text printed